Python/Product/Debugger/Debugger/DebugEngine/AD7Engine.cs

```diff
@@ -1481,5 +1481,9 @@ public sealed class AD7Engine : IDebugEngine2, IDebugEngineLaunch2, IDebugProgra
             }
             return null;
         }
+
+        internal async void RefreshThreadFrames(long threadId) {
+            await _process.GetThreadFramesAsync(threadId);
+        }
```

Comment:
At the very least, there should be a `HandleAllExceptions(...).DoNotWait()` on the task (back over in PythonTools project) in case things blow up.